rov/sensors/README.md

```diff
@@ -49,15 +52,20 @@ def Pressure():
     except Exception:
         print "Failed to Initialize BlueRobotics Bar30 Pressure Sensor"
         print "Using Mock Pressure Sensor"
+        from Pressure_Mock import Pressure as Pressure_Mock
         return Pressure_Mock()
 ```
 
-This lets you import the pressure sensor like normally: `from pressure import Pressure`
-When you attempt to initialize the pressure sensor: `Presure()`, the method is called that
-figures which one to use. First, it tries to initialize and update the data. For the pressure
-sensor, updating the data will throw an exception when it cannot write to the device. Other
-devices, like the IMU, throw an error when initializing, so an update call isn't required.
-The idea here is to fail fast, and once we know it fails, return the mock sensor.
+This lets you import the pressure sensor like normally: `from pressure import
+Pressure` When you attempt to initialize the pressure sensor: `Presure()`, the
+method is called that figures which one to use. First, it tries to initialize
+and update the data. For the pressure sensor, updating the data will throw an
+exception when it cannot write to the device. Other devices, like the IMU,
+throw an error when initializing, so an update call isn't required. The idea
+here is to fail fast, and once we know it fails, return the mock sensor. One
+important aspect, is to also put the import statements in the try/except. This
```

Comment:
Add this section about putting the _import_ statements into the try/except as well.